kvbc/include/categorization/details.h

```diff
@@ -96,5 +97,47 @@ inline void sortAndRemoveDuplicates(std::vector<std::string> &vec) {
   std::sort(vec.begin(), vec.end());
   vec.erase(std::unique(vec.begin(), vec.end()), vec.end());
 }
+// A LocalWriteBatch is used in a multithreaded context, where several jobs need to collect updates.
+// Each job gets its LocalWriteBatch.
+// then all updates are added to the Global NativeWriteBatch, which is not thread safe.
+struct LocalWriteBatch {
```

Comment:
Nice idea and a simple inteface! Allows us to have minimal changes in other places.

Reply:
thanks!